app/jobs/fs/sync_box_job.rb

```diff
@@ -4,7 +4,7 @@ def perform(box, from: Date.today - 1.week, to: Date.tomorrow)
       raise unless box.is_a?(Fs::Box)
       return unless box.syncable?
 
-      box.messages.outbox.find_each do |outbox_message|
+      box.messages.outbox.where(type: [nil, 'Message']).find_each do |outbox_message|
```

Comment:
Spravme si tu nejaky scope `except_drafts` lebo toto sa neda uplne pochopit.

Reply:
Alebo mozno `not.drafts` by sa mohlo tiez dat spravit

Reply:
pozeram ze best practice je spravit prefix `not_` a nie `except_` sorry, necham na tvoje zvazenie.